src/main/java/com/cloudant/http/interceptors/TimeoutCustomizationInterceptor.java

```diff
@@ -50,12 +42,11 @@ public TimeoutCustomizationInterceptor() {
      * @param connectTimeout value of the connect timeout
      * @param readTimeout    value of the read timeout
      */
-    public TimeoutCustomizationInterceptor(TimeoutOption connectTimeout, TimeoutOption
+    public TimeoutCustomizationInterceptor(ClientBuilder.TimeoutOption connectTimeout,
+                                           ClientBuilder.TimeoutOption
             readTimeout) {
-        this.connectTimeout = (connectTimeout != null) ? connectTimeout.asIntMillis() :
-                DEFAULT_TIMEOUT.asIntMillis();
-        this.readTimeout = (readTimeout != null) ? readTimeout.asIntMillis() : DEFAULT_TIMEOUT
-                .asIntMillis();
+        this.connectTimeout = connectTimeout.asIntMillis();
```

Comment:
I've initialized the variables to -1, and added a note about throwing an exception when null in 4b37363.  Let me know what you think.